packages/zowe-explorer/README.md

```diff
@@ -29,70 +29,56 @@ Zowe Explorer for VS Code is a Zowe&trade; component that focuses on modernizing
 
 ## Getting started
 
-Configure Zowe Explorer for VS Code, create a [team configuration file](#create-a-team-configuration-file) to connect to mainframe services, and manage secure credentials to get started using Zowe Explorer.
+Configure Zowe Explorer for VS Code and confirm you are able to access the mainframe with your _team configuration_.
 
 ### Configuring Zowe Explorer for VS Code
 
 Configure Zowe Explorer for VS Code by changing the extension settings. For more information, see [Configuring Zowe Explorer](https://docs.zowe.org/stable/user-guide/ze-install-configuring-ze).
 
-### Create a team configuration file
-
-A team configuration file stores connection information to mainframe services in _profiles_ that are easy to maintain and share with others in your organization.
-
-**Note**: Team configuration files can be used across three core components of the Zowe project, Zowe Explorer for VS Code, Zowe Explorer for IntelliJ IDEA, and Zowe CLI.
-
-1. Navigate to the **Side Bar**.
-2. Hover over the **DATA SETS**, **USS**, or **JOBS** tree view header.
-3. Click the **+** icon.
-4. Select **Create a New Team Configuration File**.
-5. If no workspace is open, a global configuration file is created. If a workspace is open, chose either a global configuration file or a project-level configuration file.
-6. Edit the configuration file to include the host and other connection information for a mainframe service, and save.
+### Confirming your team configuration
 
-   Your team configuration file appears either in your `.zowe` folder if you chose the global configuration file option, or in your workspace directory if you chose the project-level configuration file option. The notification message that displays in VS Code after configuration file creation includes the path of the created file.
-
-   You can now use all the functionalities of the extension.
-
-### Using profiles for the first time
+Team configuration stores connection information to access the mainframe. Check that you have your team configuration in place.
 
-The first time team configuration profiles are used, you are prompted for a user name and password for the profile's connection.
+If you are missing your team configuration files, `zowe.config.json` and `zowe.schema.json`, work with your administrator to set them up.
 
-The term _password_ is used loosely to represent all supported authentication secrets, such as passphrases, passtickets, Multifactor Authentication (MFA) tokens, etc.
+- Your team configuration is in place when:
+   - Selecting the **+** icon, presents options in the **Quick Pick** to edit your configuration file or, if available, use an existing profile.
```

Comment:
Should we specify where the + icon is located? I see that we mention the headers of the Data Sets/USS/Jobs tree views under the next section ("You do not have team configuration when:"). I think we can mention these headers to clarify the location of the + icon.